main-u1.cc

```diff
@@ -74,13 +76,13 @@ int main(int ac, char* av[]) {
   double rate = 0.;
   for(size_t i = gparams.icounter; i < gparams.N_meas + gparams.icounter; i++) {
     std::mt19937 engine(gparams.seed+i);
-    rate += sweep(U, engine, delta, N_hit, gparams.beta);
+    rate += sweep(U, engine, delta, N_hit, gparams.beta, xi);
     double energy = gauge_energy(U);
     double E = 0., Q = 0.;
-    energy_density(U, E, Q);
+    //~ energy_density(U, E, Q);
```

Comment:
why commented out?

Reply:
I tried to save time in computing some test cases